ui/components/component-library/banner-alert/__snapshots__/banner-alert.test.js.snap

```diff
@@ -12,13 +12,13 @@ exports[`BannerAlert should render BannerAlert element correctly 1`] = `
     />
     <div>
       <h5
-        class="box mm-text mm-banner-base__title mm-text--body-lg-medium box--flex-direction-row box--color-text-default"
+        class="mm-box mm-text mm-banner-base__title mm-text--body-lg-medium mm-box--color-text-default"
         data-testid="mm-banner-base-title"
       >
         BannerAlert test
       </h5>
       <p
-        class="box mm-text mm-text--body-md box--flex-direction-row box--color-text-default"
+        class="mm-box mm-text mm-text--body-md mm-box--color-text-default"
       >
         should render BannerAlert element correctly
       </p>
```

Comment:
## Banners

## Avatars
Before/After screencasst showing no visual regressions for all avatar components

### Before
https://github.com/MetaMask/metamask-extension/assets/8112138/56e451d0-9e40-4191-a4a3-5a54faeeef07

### After
https://github.com/MetaMask/metamask-extension/assets/8112138/33f7ce85-ba0d-45b9-af57-848e0b746f63